xml/System.Formats.Tar/TarWriter.xml

```diff
@@ -47,6 +90,30 @@
         <remarks>When using this constructor, the format of the resulting archive is <see cref="F:System.Formats.Tar.TarFormat.Pax" />.</remarks>
```

Comment:
We renamed the `TarFormat` enum to `TarEntryFormat`. I guess it's expected for mdoc to not be able to determine whenever an API name changes, which is why the crefs aren't getting updated.

Reply:
```suggestion
        <remarks>When using this constructor, the format of the resulting archive is <see cref="F:System.Formats.Tar.TarEntryFormat.Pax" />.</remarks>
```

Reply:
After seeing half of the instances of `TarFormat` being correctly changed to `TarEntryFormat`, I guess this really is a bug in mdoc. I don't know what caused it to _not_ change all of them.

Reply:
Did you see the name updated anywhere in the doc comments?

Reply:
Yes, there are many places (comments or autogenerated tags like member signatures) where the word `TarFormat` was correctly changed to `TarEntryFormat`.

Reply:
Hmm, I did a search in VS Code in this branch and didn't see any doc comments where the name was updated.